.github/workflows/Cabal.yml

```diff
@@ -46,6 +46,12 @@ jobs:
           cabal-cache-v2-${{ runner.os }}-
     - name: Build
       run: cabal v2-build $CONFIG
+      if: matrix.os != 'macOS-latest' || matrix.ghc != '9.2.2'
+      # See https://gitlab.haskell.org/ghc/ghc/-/issues/20592#note_391266 for the problem and workaround.
+      # This problem is fixed in GHC 9.2.3.
+    - name: Build with a workaround for macOS
+      run: C_INCLUDE_PATH="`xcrun --show-sdk-path`/usr/include/ffi" cabal v2-build $CONFIG
+      if: matrix.os == 'macOS-latest' && matrix.ghc == '9.2.2'
```

Comment:
Let's separate this to a separate PR. I'm planning over the holiday break to go over the CI and make it simpler and also work with supply chain tools

Reply:
I'll do that. By the way, how about changing 9.2.2 to 9.2.5? The latter is the current "recommended" version by GHCup. (I'm unsure if changing the version can delete this workaround because I don't have a mac PC, though.)

Reply:
I've created #628.

Reply:
Thank you. Merged

> By the way, how about changing 9.2.2 to 9.2.5? The latter is the current "recommended" version by GHCup.

Yup, planning work on that too.

Reply:
Hmm, 9.2.5 is not yet able to be downloaded on windows with the current CI

https://github.com/mihaimaruseac/hindent/actions/runs/3722058508/jobs/6312624631

I'll try again once refactoring the CI

Reply:
It seems it's an upstream's bug: https://github.com/haskell/actions/issues/129.

Reply:
Thanks. I'll disable it on Windows until that gets fixed